examples/icmp_response.rs

```diff
@@ -0,0 +1,268 @@
+/* ICMP example */
+use clap::{arg, Command};
+use num_derive::{FromPrimitive, ToPrimitive};
+use packetvisor::pv;
+use pnet::{
+    datalink::{interfaces, MacAddr, NetworkInterface},
+    packet::icmp::{IcmpTypes, MutableIcmpPacket},
+    packet::MutablePacket,
+    packet::{
+        arp::{ArpHardwareTypes, ArpOperations, MutableArpPacket},
+        ethernet::{EtherTypes, MutableEthernetPacket},
+        ipv4,
+    },
+    packet::{icmp, ipv4::MutableIpv4Packet},
+};
+use signal_hook::SigId;
+use std::{
+    env,
+    io::Error,
+    net::Ipv4Addr,
+    sync::atomic::{AtomicBool, Ordering},
+    sync::Arc,
+};
+
+#[derive(ToPrimitive, FromPrimitive)]
+enum Protocol {
+    ARP = 1,
+    ICMP = 2,
+    OTHER = 3,
+}
```

Comment:
same opinion in #50 

I thought enum Protocol { ARP, ICMP, OTHER } will work but I don't know the elements in enum need to have integer.
Could you check if they should have integer?